packages/harpguru-core/src/components/layout-menu/layout-menu.tsx

```diff
@@ -4,39 +4,60 @@ import { getApparatusIds } from 'harpparts'
 import { Entypo } from '@expo/vector-icons'
 
 import { MemoOptionStack } from '../option-stack'
+import { OptionLabel } from '../option-label'
+import { OptionItem } from '../option-item'
 import { MenuOpenButton } from '../menu-open-button'
 import { MenuFace } from '../menu-face'
 import { Menu } from '../menu'
-import { MenuProps } from '../../types'
+import type { MenuProps } from '../../types'
 import { colors, getSizes } from '../../styles'
 
 import { getNewHarpStrataByApparatusForDispatcher } from './utils'
 
 export const LayoutMenu = (menuProps: MenuProps): React.ReactElement => {
   const sizes = getSizes()
 
-  const items = getApparatusIds().map((id) => ({
-    label: id,
-    callbackParam: id,
-  }))
-  const useSubTitle = useCallback(() => {
+  const useTitle = useCallback(() => {
     const [activeHarpStrata] = useGlobal('activeHarpStrata')
     const {
       apparatus: { id: apparatusId },
     } = activeHarpStrata
-    return apparatusId
+    return (
+      <OptionLabel
+        name={'Tuning'}
+        isLargeTitle={true}
+        value={apparatusId}
+        alignItems={'flex-start'}
+      />
+    )
   }, [useGlobal])
   const itemTapHandler = useCallback(
     useDispatch(getNewHarpStrataByApparatusForDispatcher),
     [useDispatch]
   )
+  const useItems = useCallback(() => {
+    const [activeHarpStrata] = useGlobal('activeHarpStrata')
+    const {
+      apparatus: { id: apparatusId },
+    } = activeHarpStrata
+    const items = getApparatusIds().map((id, index) => (
+      <OptionItem
+        key={`${index}`}
+        value={id}
+        isSelected={id === apparatusId}
+        itemTapHandler={itemTapHandler}
+        callbackParam={id}
+        twoColumns={false}
+      />
+    ))
+    return items
+  }, [useGlobal])
 
   const optionStackPropsz = [
```

Comment:
However the question of creating these objects is handled in the covariant menu, the same approach should be used here.

Reply:
Done